opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -4170,7 +4170,8 @@ namespace Opm
             // Note: E100's notion of PI value phase mobility includes
             // the reciprocal FVF.
             const auto connMob =
-                mobility[ flowPhaseToEbosCompIdx(p) ].value() * fs.invB(p).value();
+                mobility[ flowPhaseToEbosCompIdx(p) ].value()
+                    * fs.invB(flowPhaseToEbosPhaseIdx(p)).value();
```

Comment:
Cool.  We probably need the same fix in the [MultisegmentWell](https://github.com/OPM/opm-simulators/blob/9552d0cc8a62f243fe47c0b0879224b47f010835/opm/simulators/wells/MultisegmentWell_impl.hpp#L4337-L4339) implementation too.

Reply:
Thanks for spotting, I will update with a fix to the MSW.